src/ucp/proto/proto_common.h

```diff
@@ -30,7 +38,9 @@ typedef struct {
     double                  overhead;      /* protocol overhead */
     size_t                  cfg_thresh;    /* user-configured threshold */
     unsigned                cfg_priority;  /* user configuration priority */
-    size_t                  fragsz_offset; /* offset of maximal fragment
+    ptrdiff_t               min_sz_offset; /* offset of minimal size in
+                                              uct_iface_attr_t */
+    ptrdiff_t               fragsz_offset; /* offset of maximal fragment
```

Comment:
minor: `fragsz_offset` name is not consistent with `min_sz_offset` (missing `_` after `frag`)

Reply:
fixed